src/Widgets/SliderWidget/SliderWidgetEditingConfig.ts

```diff
@@ -0,0 +1,50 @@
+import { provideEditingConfig } from 'scrivito'
+import { SliderWidget } from './SliderWidgetClass'
+import { SlideWidget } from '../SlideWidget/SlideWidgetClass'
+import thumbnail from './thumbnail.svg'
+
+provideEditingConfig(SliderWidget, {
+  title: 'Slider',
+  thumbnail,
+  attributes: {
+    autoPlay: {
+      title: 'Auto play?',
+      description: 'Default: No',
+    },
+    controls: {
+      title: 'Show controls?',
+      description: 'Shows arrows and indicators. Default: Yes',
+    },
+    margin: {
+      title: 'Margin',
+      description: 'Outer space to next item (bottom). Default: "mb-4"',
+    },
+    minHeight: {
+      title: 'Minimum height (in px)',
+      description: 'Default: 400',
```

Comment:
* [ ] FYI we could explain what this is for: to prevent jumping content if individual slides have different height (but better wording than mine).